resources/public/api/conf/3.0/schemas/retrieve/calculation/employmentAndPensionsIncome.json

```
@@ -1,11 +1,514 @@
+                    "type": "number",
+                    "multipleOf": 0.01,
+                    "minimum": 0,
+                    "maximum": 99999999999.99,
+                    "example": "2000.98"
+                  },
+                  "taxPaid": {
+                    "description": "Tax paid on benefits from employment finance retirement scheme. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                    "type": "number",
+                    "multipleOf": 0.01,
+                    "minimum": 0,
+                    "maximum": 99999999999.99,
+                    "example": "2000.98"
+                  },
+                  "taxTakenOffInEmployment": {
+                    "description": "A boolean to identify whether tax was taken off in employment.",
+                    "type": "boolean"
+                  }
+                }
+              },
+              "redundancyCompensationPaymentsOverExemption": {
+                "description": "Object detailing any redundancy compensation payments over exemption.",
+                "type": "object",
+                "properties": {
+                  "amount": {
+                    "description": "The amount from Redundancy Compensation Payments Over Exemption. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                    "type": "number",
+                    "multipleOf": 0.01,
+                    "minimum": 0,
+                    "maximum": 99999999999.99,
+                    "example": "2000.98"
+                  },
+                  "taxPaid": {
+                    "description": "Tax Paid on Redundancy Compensation Payments Over Exemption. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                    "type": "number",
+                    "multipleOf": 0.01,
+                    "minimum": 0,
+                    "maximum": 99999999999.99,
+                    "example": "2000.98"
+                  },
+                  "taxTakenOffInEmployment": {
+                    "description": "A boolean to identify whether tax was taken off in employment.",
+                    "type": "boolean"
+                  }
+                }
+              },
+              "redundancyCompensationPaymentsUnderExemption": {
+                "description": "Object detailing any redundancy compensation payments under exemption",
+                "type": "object",
+                "properties": {
+                  "amount": {
+                    "description": "The amount of Redundancy Compensation Payments Under Exemption. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+                    "type": "number",
+                    "multipleOf": 0.01,
+                    "minimum": 0,
+                    "maximum": 99999999999.99,
+                    "example": "2000.98"
+                  }
+                }
+              }
+            }
```

This should be the end of `lumpSums` and the start of `studentLoans` (which should be at the same level as `lumpSums`):
```suggestion
            }
          }
        },
        "additionalProperties": false,
        "required": [
          "totalLumpSum",
          "lumpSumsDetail"
        ]
      }            
```